.github/workflows/ci.yml

```
@@ -49,14 +47,18 @@ jobs:
       - uses: actions/checkout@v2
       
       #- name: install utils
-      #  run: apt-get update && apt-get install -y openssh-client postgresql-client curl
+      #  run: |
+      #    sudo apt-get update
+      #    sudo apt-get install -y openssh-client postgresql-client curl
         
       - name: db setup
         uses: docker://pecan/db:latest
 
       - name: create account
-        uses: docker://pecan/bety:latest
-        args: user carya illinois "Carya Demo User" carya@example.com 1 1
+        uses: docker://pecan/bety:develop
+        with:
+          # entrypoint: /home/bety/script/betyuser.sh
+          args: user carya illinois "Carya Demo User" carya@example.com 1 1
```

FWIW, I'm currently testing this out on a personal branch of this.

```suggestion
          entrypoint: /bin/bash
          args: /home/bety/script/betyuser.sh carya illinois "Carya Demo User" carya@example.com 1 1
```

---

Though, I guess setting `entrypoint: /home/bety/script/betyuser.sh` might be a more straightforward version of the same thing.